### ISSUE TITLE
Error when activating theme (Aseprite v1.2.8)

### ISSUE THREAD

**Issue by JaydedCompanion** [el-falso/monaki-theme#1]
Hello! I recently purchased this theme on your Itch.io page, but I'm getting the following error when I try to activate it. I'm currently using Aseprite v1.2.8 on OS X Mojave (10.14)

```
A problem has occurred.

Details:
Part <background part='timeline_focused' ...> was not found in '/Users/.../Library/Application Support/Aseprite/extensions/monaki-theme/./theme.xml'
```

**Comment by el-falso** [comment]
Seems like an update of Aseprite is causing this. Can't find "timeline_focused" in my .xml either, but in Aseprites standard theme.
Going to drop a quickfix soon.

But somehow the theme is working for me using Aseprite v1.2.10-beta4 on Windows 10

**Comment by el-falso** [comment]
Dropped a fix with Monaki v1.3
